implementations/build-topaz.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e # make script fail on first error
+SCRIPT_PATH=`dirname $0`
+source $SCRIPT_PATH/script.inc
+
+INFO Build Topaz
+pushd $SCRIPT_PATH
+  git clone git://github.com/topazproject/topaz
+  pushd topaz
+    git checkout tim/features
+    pip install invoke
+    export TEST_TYPE=translate-jit-notest
+    invoke travis.install_requirements
+    invoke travis.run_tests
+  popd
+
+  cat <<EOF>topaz.sh
+#!/bin/bash
+unset CDPATH
+pushd \`dirname \$0\`
+BINPATH="\$(pwd)"
+popd
```

Comment:
How about simply having a checked-in file `implementations/topaz.sh` like e.g. [jruby.sh](https://github.com/smarr/are-we-fast-yet/blob/master/implementations/jruby.sh) ?